documentation/modules/con-securing-kafka-authorization.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-access.adoc
+
+[id='con-securing-kafka-authorization-{context}']
+= Kafka authorization
+
+You can configure authorization for Kafka brokers using the `authorization` property in the `Kafka.spec.kafka` resource.
+If the `authorization` property is missing, no authorization is enabled.
```

Comment:
Let's make this more clear by explaining the consequences of using no authhorization.

```suggestion
If the `authorization` property is missing, no authorization is enabled and all clients are allowed to do anything.
```

Reply:
..no authorization is enabled and clients have no restrictions.